crypto/base/include/privmx/crypto/ecc/ExtKey.hpp

```diff
@@ -78,6 +79,9 @@ inline std::string ExtKey::getPublicPartAsBase58() const {
 }
 
 inline PrivateKey ExtKey::getPrivateKey() const {
+    if (!_is_private) {
+        throw DeriveFromPublicKeyNotImplementedException();
```

Comment:
Change exception name for all methods, e.g.:
```suggestion
        throw ExtKeyDoesNotHoldPrivateKeyException();
```